### ISSUE TITLE
Difficulty getting pre-trained models to work (e.g. mobilenet v2)

### ISSUE THREAD

**Issue by Nacho114** [epfml/disco#430]
I am trying to get mobilenet v2 to work on disco, but I am running into some issues (see TL;DR and the end).

The easiest solution would be to get mobilenet v2 via [npm](https://www.npmjs.com/package/@tensorflow-models/mobilenet), however this version only allows for inference and it cannot be used for training.

For mobile net v1 we loaded an existing tfjs model as is done in this [thread](https://stackoverflow.com/questions/59930486/how-to-tf-loadlayersmodel-of-mobilenet-v2).

The issue with mobilenet v2, is that in [TFhub](https://tfhub.dev/google/imagenet/mobilenet_v2_035_224/classification/5) all tfjs models are converted from TF models. The issue is that such [conversions](https://github.com/tensorflow/tfjs/tree/master/tfjs-converter) yield only graph models, these can only be used for inference and not for training. 

If we convert from a keras model, then we can get a tfjs layers model which we can then use for training. However this poses two problems:

1. I am getting a `TypeError: Only HTTP(S) protocols are supported` when loading a layers model through the local file system, this used to work before (and it works on a stand alone project that I tested), I suspect that this might no longer work due to importing both `tfjs` and `tfjs-node`. Note that due to this import issue we are also unable to save models currently as seen in #363, this might also be the issue with #416. Loading from the local system is not a good solution in any case, since it's not possible in tfjs browser which has only access to indexedDB, it is however an easy way to test existing models in node, so it would good to fix it for this alone.

2. To load a model via url as was done for mobilenet v1 we need to serve the model in our server via our server.

TL;DR

1. If we want to use a TF model for training in tfjs, then we need to first convert it. This will only be possible if the source model is a keras model as we need a layers models in the output. I could not find a hosted mobile net v2 with the layers model format (probably due to the fact that everyone does inference on the browser). I have converted a keras model, but getting it to run in disco is difficult to due 2.

2. Loading models from the local file system with loadLayersModel appears to be broken in discojs (maybe this should be a separate issue). Related to #363 and #416.

4. If we want to use pre-trained models, we will need to serve them in the server via the appropriate http request handler.

**Comment by Nacho114** [comment]
In the meantime I can build mobilenet v2 from [scratch](https://towardsdatascience.com/building-mobilenet-from-scratch-using-tensorflow-ad009c5dd42c), so without pre-training

**Comment by martinjaggi** [comment]
maybe we can get it from keras https://keras.io/api/applications/mobilenet/

then use `savemodel` into a `tfLayerModel` format?

**Comment by Nacho114** [comment]
Replaced by #446
